### PR TITLE
Fixing typos in 2 files

### PR DESCRIPTION
**Pull request by didier-durand**
Hi,
as title says, fixing a couple of typos. See commit diffs for details.
Cheers
Didier

---

> [!NOTE]
> Fixes two documentation typos: “ML Layesrs” -> “ML Layers” and “finetuned” -> “fine-tuned”.
> 
> - **Docs**:
>   - `quarto/contents/core/benchmarking/benchmarking.qmd`: Fix node label typo from `ML Layesrs` to `ML Layers`.
>   - `quarto/contents/core/ondevice_learning/ondevice_learning.qmd`: Standardize hyphenation from `finetuned` to `fine-tuned`.
> 
> <sup>Written by [Cursor Bugbot](https://cursor.com/dashboard?tab=bugbot) for commit 8e0838d45f75bb96de5e4f00297a35d489375c75. This will update automatically on new commits. Configure [here](https://cursor.com/dashboard?tab=bugbot).</sup>

### CODE REVIEW

**Comment by didier-durand** [comment]
Hi, that's fine. Thks!
